mne_qt_browser/_pg_figure.py

```diff
@@ -1578,7 +1578,7 @@ def __init__(self, main, title='Settings', **kwargs):
                 '<i>(Those methods are adapted from '
                 'pyqtgraph)</i><br>'
                 'Default is "peak".')
-        self.ds_method_cmbx.addItems(['subsample', 'mean', 'peak'])
+        self.ds_method_cmbx.addItems(['mean', 'subsample', 'peak'])
```

Comment:
this is unrelated right?

Reply:
clearly unrelated to the crash. Attended here: 798c39d31e98c1ac1f64866f37c8ead470a2c994